content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -32,14 +32,14 @@ const LAYOUT_VARIANTS = {
   "dev-test-feeds": "Stress testing for slow feeds",
 };
 
-export class ToggleSpocButton extends React.PureComponent {
+export class ToggleStoryButton extends React.PureComponent {
```

Comment:
This is now generic for stories of all types, recs and spocs. So needed a name change.